client/reader/stream/x-post.jsx

```diff
@@ -149,8 +168,27 @@ class CrossPost extends PureComponent {
 						}
 					{ this.getDescription( post.author.first_name ) }
 				</div>
-			</Card> );
+				{ feedId && <QueryReaderFeed feedId={ +feedId } includeMeta={ false } /> }
```

Comment:
`shouldFeedBeFetched` Right you are! I didn't realize it checked both that it wasn't queued _and_ that we dont already have it!

```js
/**
* Returns true if we should fetch the feed
*
* @param  {Object}  state  Global state tree
* @param  {Number}  feedId The feed ID
* @return {Boolean}        Whether feed should be fetched
*/

export function shouldFeedBeFetched( state, feedId ) { 
  return ! state.reader.feeds.queuedRequests[ feedId ] && // not currently queued
    ! state.reader.feeds.items[ feedId ]; // not currently loaded
}
```